File: 1980-find-unique-binary-string/1980-find-unique-binary-string.cpp
// m3hu1
class Solution {
public:
    string findDifferentBinaryString(vector<string>& nums) {
        int i =0; 
        string ans;  
        for(string &str: nums){
            ans.push_back(str[i]=='0'?'1':'0');
            i++;  
        }
        return ans; 
    }
};

File: 1980-find-unique-binary-string/1980-find-unique-binary-string.go
func findDifferentBinaryString(nums []string) string {
	ans := make([]byte, len(nums))
    
	for i := 0; i < len(nums); i++ {
		if nums[i][i] == '0' {
			ans[i] = '1'
		} else {
			ans[i] = '0'
		}
	}

	return string(ans)
}